frontend/modules/recipe/components/Recipe.js

```diff
@@ -126,17 +126,12 @@ var RecipeScheme = injectIntl(React.createClass({
         description: 'minutes',
         defaultMessage: 'minutes'
       },
-      photo_placeholder: {
-        id: 'recipe.photo_placeholder',
-        description: 'Photo placeholder',
-        defaultMessage: 'Generic placeholder thumbnail'
-      }
     });
 
     return (
       <div className="recipe-details">
         <h1 className="title hidden-xs">{this.props.data.title}</h1>
```

Comment:
Good point, but now that I've had a good look, I'm not convinced the current layout makes a lot of sense. If the image exists, we show the image twice, which looks kinda weird. And if there is no image, then we show the title of the recipe in a different place and a useless placeholder. I feel like we should either just show the large image first and no first header or thumbnail. Or just the thumbnail (with a placeholder if no image) and have the green title always be the header. Or maybe the large image appears under the green title? 

I'm going to remove the first title if the image doesn't exist for the time being, and we can open a separate issue to see if we can improve this page.